### PR TITLE
[IOTDB-920] Disable insert row thats only contains time/timestamp column

### PR DESCRIPTION
**Pull request by yuqi1129**

### CODE REVIEW

**Comment by HTHou** [comment]
This pr is changing too many files. Please close it, pull master branch again, do the modification and re-submit a pr again. Thanks.

**Comment by yuqi1129** [comment]
@HTHou 
Yes, i can't figure out why it involved so many files, i will update the code today as it's too late last night when i make the PR

File: server/src/test/java/org/apache/iotdb/db/integration/IOTDBInsertIT.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
+package org.apache.iotdb.db.integration;
```
Comment (mychaow) [inline server/src/test/java/org/apache/iotdb/db/integration/IOTDBInsertIT.java:1]: add license.

File: server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java [hunk 2]
```diff
@@ -145,9 +145,13 @@
 import org.apache.iotdb.tsfile.utils.Pair;
 import org.apache.iotdb.tsfile.write.schema.MeasurementSchema;
 import org.apache.iotdb.tsfile.write.writer.RestorableTsFileIOWriter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class PlanExecutor implements IPlanExecutor {
 
+  private static final Logger LOG = LoggerFactory.getLogger(PlanExecutor.class);
```
Comment (mychaow) [inline server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java:153]: We usually use logger, maybe you could rename it.

File: server/src/test/java/org/apache/iotdb/db/integration/IOTDBInsertIT.java (hunk 1 above)
Comment (yuqi1129) [reply server/src/test/java/org/apache/iotdb/db/integration/IOTDBInsertIT.java:1]: OK

File: server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java (hunk 2 above)
Comment (yuqi1129) [reply server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java:153]: Sure, if it's the custom

**Comment by jixuan1989** [comment]
Hi @yuqi, thanks for your contribution!

Haonan mentioned something about the code format style in another PR: we use double blank spaces rather than 4 blank spaces (or tab with 4 blank spaces) for Java Files.

If you use a different code format style, then someone may reformat the file in the future and there will be a lot of meaningless changes.

Read https://iotdb.apache.org/Development/ContributeGuide.html : 
```
https://iotdb.apache.org/Development/ContributeGuide.html
```

**Comment by yuqi1129** [comment]
@jixuan1989 
OK, I will read the guild more carefully and continue to work, thanks for your help

**Review by qiaojialin — APPROVED** [review]
LGTM

**Review by HTHou — APPROVED** [review]

**Pull request merged**